docs/docs/dev-docs/api-contracts/Analytics.md

```diff
@@ -0,0 +1,78 @@
+# Admin Analytics
+
+## Helper types
+
+```ts
+interface time {
+    days : number;
+    hours : number;
+    minutes: number;
+}
+```
+```json
+ //idk what the ts of this is
+ // This is a GroupedValue format
+ {
+    "key" : number,
+    ...
+ }
+```
+
+# Predicted time to complete a dispute
+
+- **Endpoint:** `GET /time/estimation`
+- **Headers:**
+  - `Authorization: Bearer <JWT>`
+- **Note:** only available to admins
+
+```ts 
+//returns
+//if resolved disputes exist
+interface ResponseResolvedDisputes {
+    data : time
+}
+//otherwise
+interface error {
+    error : string
+}
+```
```

Comment:
very small nit, but you can just say
```ts
type ResponseResolvedDisputes = number;
```
We assume the result type for everything